controllers/nicclusterpolicy_controller.go

```diff
@@ -53,6 +53,7 @@ type NicClusterPolicyReconciler struct {
 	stateManager state.Manager
 }
 
+// In case of adding support for additional types, also update in getSupportedGVKs func in pkg/state/state_skel.go
```

Comment:
An Idea that popped into mind:

when creating/updating objects, check that an object is in supported GVKs, if it is not then emit a warning in logs that it will not be deleted WDYT ?

we will then have also a visual queue during runtime, WDYT ?

Reply:
Done